repo-scripts/size-analysis/analysis-helper.ts

```diff
@@ -240,30 +257,47 @@ export function extractDeclarations(
       if (symbol && symbol.valueDeclaration) {
         const importFilePath = symbol.valueDeclaration.getSourceFile().fileName;
         // import { a, b } from '@firebase/dummy-exp'
+        // import {a as A, b as B} from '@firebase/dummy-exp'
         if (
           node.importClause.namedBindings &&
           ts.isNamedImports(node.importClause.namedBindings)
         ) {
           node.importClause.namedBindings.elements.forEach(each => {
-            const symbolName: string = each.name.getText(sourceFile);
-            symbolLocation.set(symbolName, importFilePath);
+            const symbolName: string = each.name.getText(sourceFile); // import symbol current name
+            importSymbolCurrentNameToModuleLocation.set(
+              symbolName,
+              importFilePath
+            );
+            // if imported symbols are renamed, insert an entry to importSymbolCurrentNameToOriginalName Map
+            // with key the current name, value the original name
+            if (each.propertyName) {
+              importSymbolCurrentNameToOriginalName.set(
+                symbolName,
+                each.propertyName.getText(sourceFile)
+              );
+            }
           });
-          // import * as fs from 'fs'
+
+          // import * as fs from 'fs
         } else if (
           node.importClause.namedBindings &&
           ts.isNamespaceImport(node.importClause.namedBindings)
         ) {
           const symbolName: string = node.importClause.namedBindings.name.getText(
             sourceFile
           );
-          symbolLocation.set(symbolName, importFilePath);
+          declarations.variables.push(symbolName);
```

Comment:
IIUC, this assumes `fs` is part of the public API, but it might not be. Consider the following case:
```
import * as fs from 'fs';

export declare const aVariable: fs.someType;
```